Description: pass PYTHONPATH when building HTML doc
Author: Sandro Tosi <morph@debian.org>
Last-Update: 2023-10-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -6,6 +6,7 @@
 SPHINXBUILD   = sphinx-build
 PAPER         =
 BUILDDIR      = _build
+PYTHONPATH    = .
 
 # Internal variables.
 PAPEROPT_a4     = -D latex_paper_size=a4
@@ -37,7 +38,7 @@
 	-rm -rf $(BUILDDIR)/*
 
 html:
-	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	PYTHONPATH=$(PYTHONPATH) $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
